array.go

```diff
@@ -234,14 +235,14 @@ func (a *Array) Consolidate(config *Config) error {
 // Vacuum cleans up the array, such as consolidated fragments and array metadata.
 func (a *Array) Vacuum(config *Config) error {
 	if config == nil {
-		return fmt.Errorf("Config must not be nil for Vacuum")
+		return errors.New("Config must not be nil for Vacuum")
 	}
 
 	curi := C.CString(a.uri)
 	defer C.free(unsafe.Pointer(curi))
 	ret := C.tiledb_array_vacuum(a.context.tiledbContext, curi, config.tiledbConfig)
 	if ret != C.TILEDB_OK {
-		return fmt.Errorf("Error vacuumimg tiledb array: %s", a.context.LastError())
+		return fmt.Errorf("error vacuumimg tiledb array: %w", a.context.LastError())
```

Comment:
Might as well fix the typo while we're at it.
```suggestion
		return fmt.Errorf("error vacuuming tiledb array: %w", a.context.LastError())
```